.github/workflows/on_push.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
     timeout-minutes: 10
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
I'm updating these, although I'm not sure we actually need to check out the repo in this file at all.